src/server/services/procedures/daily-word-guess/daily-word-guess.js

```diff
@@ -0,0 +1,157 @@
+const { RPCError } = require('../utils');
+const WordGuess = require('../word-guess/word-guess');
+const GetStorage = require('./storage');
+const logger = require('../utils/logger')('daily-word-guess');
+
+/**
+ * Wordle-like game with daily challenges
+ * 
+ * @alpha
+ * @service
+ * @category Games
+ */
+const DailyWordGuess = {};
+
+DailyWordGuess._collection = GetStorage().usedWords;
+DailyWordGuess._dailyWord = '';
+DailyWordGuess._dailyWordDate = null;
+DailyWordGuess._states = {};
+
+/**
+ * Guess the word. Returns a list where each item is the feedback for
+ * the corresponding character. Feedback is a "3" if the character is
+ * correct, "2" if it is correct but in the wrong place, and "1" if the
+ * letter is not present in the word.
+ *
+ * @param {BoundedString<5,5>} word Guess for this round
+ */
+DailyWordGuess.guess = async function (word) {
+    word = word.toLowerCase();
+    let realWord = await DailyWordGuess._getDailyWord();
+
+    let prevState = DailyWordGuess._getUserState(this.caller.clientId);
```

Comment:
It might be nice to only use the client ID if the user is not logged in (otherwise use the username)

Reply:
Or you could try to save it using both so the user is remembered if he/she logs in, etc.